docs/source/reference/tools/xtce_generator.rst

```diff
@@ -0,0 +1,81 @@
+.. _xtce_generator:
+
+Generating Telemetry XML with Python Script
+===========================================
+
+This document provides steps and information on how to use
+`xtce_generator_template.py` script as a base for users to generate
+telemetry XML files using the TelemetryGenerator class. The script is designed to
+simplify the process of creating telemetry definitions for various packet types.
+
+The script is located in the `tools/xtce_generation` directory. The script is called
+`xtce_generator_template.py`. The script is a ``template`` that can be modified to
+generate telemetry XML files for different packet types. Your new file should be
+called `xtce_generator_yourinstrument.py`.
+An example of how to use the script is `xtce_generator_codice.py` which is also
+located in the `tools/xtce_generation` directory.
+
+Prerequisites
+-------------
+
+Before you begin, ensure you have the following:
+
+- Python installed (version 3.9 or higher)
+- The required Python packages installed:
+    - pathlib
+    - tools.xtce_generation.telemetry_generator
+
+How to Use
+----------
+
+Define the instrument name in the `main()` function by setting the `instrument_name`
+variable to the name of your instrument.
+
+::
```

Comment:
```suggestion
variable to the name of your instrument::
```

I think you can eliminate this? Or alternatively you should add a `.. code::` or some qualifier, the double colons feel empty hanging out there otherwise. (Also to the ones below this)